candid/evm_rpc.did

```diff
@@ -201,26 +207,26 @@ type ValidationError = variant {
 service : {
   authorize : (principal, Auth) -> ();
   deauthorize : (principal, Auth) -> ();
-  eth_feeHistory : (RpcSource, FeeHistoryArgs) -> (MultiRpcResult);
-  eth_getBlockByNumber : (RpcSource, BlockTag) -> (MultiRpcResult_1);
-  eth_getLogs : (RpcSource, GetLogsArgs) -> (MultiRpcResult_2);
+  eth_feeHistory : (RpcSource, FeeHistoryArgs) -> (MultiFeeHistoryResult);
+  eth_getBlockByNumber : (RpcSource, BlockTag) -> (MultiGetBlockByNumberResult);
+  eth_getLogs : (RpcSource, GetLogsArgs) -> (MultiGetLogsResult);
   eth_getTransactionCount : (RpcSource, GetTransactionCountArgs) -> (
-      MultiRpcResult_3,
-    );
-  eth_getTransactionReceipt : (RpcSource, text) -> (MultiRpcResult_4);
-  eth_sendRawTransaction : (RpcSource, text) -> (MultiRpcResult_5);
+    MultiGetTransactionCountResult
+  );
+  eth_getTransactionReceipt : (RpcSource, text) -> (MultiGetTransactionReceiptResult);
+  eth_sendRawTransaction : (RpcSource, text) -> (MultiSendRawTransactionResult);
   getAccumulatedCycleCount : (nat64) -> (nat) query;
   getAuthorized : (Auth) -> (vec text) query;
   getNodesInSubnet : () -> (nat32) query;
   getOpenRpcAccess : () -> (bool) query;
   getProviders : () -> (vec ProviderView) query;
   registerProvider : (RegisterProviderArgs) -> (nat64);
-  request : (JsonRpcSource, text, nat64) -> (Result_6);
-  requestCost : (JsonRpcSource, text, nat64) -> (Result_7) query;
+  request : (JsonRpcSource, text, nat64) -> (RequestResult);
+  requestCost : (JsonRpcSource, text, nat64) -> (RequestCostResult) query;
```

Comment:
understanding questions: what are those 2 methods `request` and `requestCost`?

Reply:
`request()` is the generic JSON-RPC endpoint (to cover everything outside of the 6 Candid-RPC methods), and `requestCost()` calculates the number of cycles required for a given request. This is explained in more detail in the design document.